webapp/components/wpt-results.js

```diff
@@ -441,8 +450,22 @@ class WPTResults extends WPTColors(WPTFlags(SelfNavigation(LoadingState(TestRuns
     return null;
   }
 
-  computeTestCanBeRunOnW3C(testType) {
-    return testType !== 'wdspec';
+  computeTestW3CURL(testType, path) {
+    if (testType === 'wdspec') {
+      return;
+    }
+    return new URL(`${this.scheme}://w3c-test.org${path}`);
+  }
+
+  computeTestRefURL(testType, path, manifest) {
+    if (!this.showTestRefURL || testType !== 'reftest') {
+      return;
+    }
+    const item = Object.values(manifest.items['reftest']).find(v => v.find(i => i[0] === path));
+    // In item[0], the 2nd item is the refs array, and we take the first ref (0).
```

Comment:
If you know where it's generated, can you point to the Python code generating the manifest that this needs to be in sync with? Maybe @Hexcles can find it quickly.